app/models/medicaid_application_member_attributes.rb

```diff
@@ -45,6 +46,14 @@ def requesting_health_insurance_key
   end
 
   def under_21?
-    member.age < 21
+    if member.birthday.present?
+      member.age < 21
+    end
+  end
+
+  def over_21_or_21?
```

Comment:
Why do we need this? Since under_21? will return `nil` if the birthday isn't present, `yes_or_no` would give the correct output, ya?

Reply:
I was having a hard time sticking with `yes_or_no` here because if we have no birthday we want to return `nil` for both, and `yes_or_no` would always have us returning "Yes" for one of the options.

Reply:
Oh RIGHT got it

Hmm, thoughts:
```
if member.birthday.present?
  member_attributes[:"#{position}_member_birthday"] => member.formatted_birthday,
  member_attributes[:"#{position}_member_under_21_#{yes_no(under_21?)}"] => "Yes",
end
```

and then nuke the presence conditional from `under_21?`